update tokenomics (#3355)
hyperledger-labs/splice: docs/src/background/tokenomics/overview_tokenomics.rst

be shared between the application provider and application user, based
on a given ``weight`` for each. The general pattern for this is:

-  A list of beneficiaries, each with a ``weight``, is provided. The weights sum up to ``1.0``.

-  Later processing creates a separate contract for each beneficiary and weight pair,
   setting the contract's ``beneficiary`` and ``weight`` fields accordingly.

Beneficiaries are discussed further in the following sections.

`CIP-0078 <https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0078/cip-0078.md>`__ eliminated
almost all fees for Canton Coin transfers and locks so unfeatured applications no longer receive any reward.
The holding fee remains but its behavior changed so that no holding fees are charged when using a coin as an input to a transfer.

The holding fees is a fixed fee, per separate coin contract (UTXO) per unit
of time, that is independent of the coin amount.  It promotes merging of CC to reduce network storage use by incentivizing merging or removal of dust coins.
Holding fees are not charged when transferring coins but only explicitly on expired coin contracts via the ``Amulet_Expire`` choice,

> This makes accounting for holding fees simple as the value of the coin contract is charged as a holding fee.

Reviewer: ```suggestion
This makes accounting for holding fees simple as the value of the coin contract is constant and independent of the holding fee.
```
